# The unique name of the task
string task_name
# The full name of the actionserver
string task_server_name
# true for a registration
# false for a unregistration
bool reg
